[FCE-576]: Update quick setup docs to use proper names

MiloszFilimowski opened the pull request:
## Description

- Updated docs to work out of the box

mironiasty commented on guide/index.md:
```diff
@@ -6,81 +6,218 @@ sidebar_position: 1
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  tracksContainer: {
+    flex: 1,
+    alignItems: "center",
+  },
+  videoElement: {
+    width: "85%",
+    aspectRatio: 1,
+    backgroundColor: "gray",
+    marginVertical: 16,
+  },
+});
```
I would prefer to avoid using styles in example code. Whenever I use example, I prefer to copy part of code and styles make it more difficult

karkakol replied:
This must stay, or at least dimension style of `VideoRenderer`, otherwise they will not have size and nothing would  display.

MiloszFilimowski replied:
It's all in one snippet that you can copy, otherwise those snippets are useless and won't work I think this is the main goal of a 5 minute setup so that everything works.

mironiasty commented on guide/index.md:
````diff
@@ -6,81 +6,218 @@ sidebar_position: 1
 ```ts
 const response = await fetch(
-  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?room=ROOM_NAME&user=USER_NAME`,
+  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/*roomName*/users/*username*`,
````
Let's use new params: 
```suggestion
  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?roomName=*roomName*&userName=*username*`,
```

karkakol replied:
Does it work?
I tried it yestarday and params passed like this didn't work :/

MiloszFilimowski replied:
Are you sure this is the new way? We have slashes both on staging and prod.

mironiasty replied:
slashes are now deprecated. But also we should update `participantName` back to peerName

MiloszFilimowski replied:
fixed

mironiasty commented on guide/index.md:
````diff
@@ -6,81 +6,218 @@ sidebar_position: 1
-To start streaming, you have to prepare your camera and join room:
+:::danger
 
-```ts
-import { joinRoom, useCamera } from "@fishjam-cloud/react-native-client";
+If you want to use a device camera, you must first request permission. Keep in mind that this won't work on iOS Simulator.
+Check [this `guide`](./react-native/installation#step-2-configure-app-permissions) for more information.
 
+:::
````
I would split it into two boxes: one about permissions, and one about iOS simulator. These are two different things.

mironiasty commented on guide/index.md:
````diff
@@ -6,81 +6,218 @@ sidebar_position: 1
+});
+```
+
+### Full example
````
I'm not sure if this is needed. How am I suppose to use this example, when I try to integrate Fishjam Cloud with my app?

I think it would make more sense to link to our example app (or create some event simpler app that works more or less like the one below?)

karkakol replied:
Imo this is needed. At least when I search packages to do stuff i see that provided ready example does not work i do not even bother to use it in my app.

MiloszFilimowski replied:
This is the only way to see everything in context and show the user how to use a minimal example. Personally I much prefer having one snippet that I can analyze and copy-paste. 

You can replace any of your app's screens and when you paste the example it will just work. Our example is too difficult to set up for a quick check.

mironiasty commented on guide/index.md:
```diff
@@ -6,81 +6,218 @@ sidebar_position: 1
-  });
-
-  return <Button onPress={startStreaming}>Start Streaming</Button>
+  return <Button title={"Start Streaming"} onPress={startStreaming} />;
```
nit: with our styling standards it should be:
```suggestion
  return <Button title="Start Streaming" onPress={startStreaming} />;
```

karkakol reviewed (changes requested):
Maybe it would be a good idea to show from where take room manager url:
<img width="920" alt="image" src="https://github.com/user-attachments/assets/d2c239d9-eaea-48d1-b36a-384169eb5eff">
Even for me during testing that was not clear.

<img width="899" alt="image" src="https://github.com/user-attachments/assets/2dd32f87-2753-4258-9811-1f06400e0822">
This section moves user to `documentation/guide/react-native/installation`
There is no information about how to request in runtime permissions, and this can be done easily by adding: 

```
import { useEffect } from 'react';
import { Platform, PermissionsAndroid } from 'react-native';

export function usePermissionCheck() {
  useEffect(() => {
    async function request() {
      if (Platform.OS === 'ios') {
        return;
      }
      try {
        await PermissionsAndroid.requestMultiple([
          PermissionsAndroid.PERMISSIONS.CAMERA,
          PermissionsAndroid.PERMISSIONS.RECORD_AUDIO,
          PermissionsAndroid.PERMISSIONS.POST_NOTIFICATIONS,
        ]);
      } catch (err) {
        console.warn(err);
      }
    }

    request();
  }, []);
}

```
Or any other easy way.
Another thing is 
`function StartStreamingButton({ roomName, userName }) {`
could be changed to 
`function StartStreamingButton({ roomName, userName }: { roomName: string, userName: string }) {`
to avoid IDE errors.

Here:
```
  const startStreaming = useCallback(async () => {
        const response = await fetch(
            `https://k8s-cloud.fishjam.work/api/v1/connect/6d8092f65b32440b9f4c4f29aec3da09/room-manager/${roomName}/users/${userName}`,
        );
        const { url, peerToken } = await response.json();

        await prepareCamera({ cameraEnabled: true });

        await joinRoom(url, participantToken);
    }, []);
```
Should be changed to:
```
  const startStreaming = useCallback(async () => {
        const response = await fetch(
            `https://k8s-cloud.fishjam.work/api/v1/connect/6d8092f65b32440b9f4c4f29aec3da09/room-manager/${roomName}/users/${userName}`,
        );
        const { url, participantToken } = await response.json();

        await prepareCamera({ cameraEnabled: true });

        await joinRoom(url, participantToken);
    }, []);
```
unless request response would change from room manager.

MiloszFilimowski commented:
@karkakol 
- Let's not a screenshot for now and let's discuss later if we need it
- Added the permissions step to #Installation doc
- Added missing types
